hphp/hack/test/typecheck/xhp_modifier/xhp_modifier_no_dashes.php

```diff
@@ -0,0 +1,4 @@
+<?hh // strict
+
+# this is not allowed
+xhp class Foo-bar {}
```

Comment:
Great test! Minor nit: I'd prefer `//` style comments for consistency with our other tests.

Reply:
Will fix. This our internal style and old habits die hard :)